mousebender/simple.py

```diff
@@ -90,16 +88,39 @@ def parse_repo_index(html):
     return parser.mapping
 
 
-@dataclasses.dataclass(frozen=True)
```

Comment:
If you're going to do this then use [attrs](https://pypi.org/project/attrs/).

Reply:
Please don't make us vendor `attrs` in pip. :(

Reply:
Are you actually wanting to use this code _that_ soon in pip? If you're going to pull it in really soon then that's fine. But if this is more of  a long-term thing then we can remove the use of attrs later once everyone is happy with how the API and code function.

IOW I would rather make the "get off of attrs" a chore when it truly needs to be worried about and make this shift over to older Python 3 versions easier/smoother.

Reply:
Fair enough. :)